.github/workflows/test.yml

```
@@ -31,4 +31,7 @@ jobs:
         go-version: 1.16
 
     - name: Test
+      env:
+        TF_TOKEN: ${{ secrets.OSS_TF_TOKEN }}
```

```suggestion
        TF_TOKEN: ${{ secrets.TF_TOKEN_OSS }}
```

Suffixes over prefixes in most cases, so that like variables sort together